README.md

```diff
@@ -70,6 +70,11 @@ Update your local `hosts` file with the remote server's ip address
 remote-server-ip-address (maintain other default options)
 ```
 
+If you are on Ubuntu change `deploy_group` variable in the [configuration file](https://github.com/consul/installer/blob/1.4.1/group_vars/all) to
```

Comment:
Do you think it makes sense to move these lines to the "Configuration Variables" section? :thinking:

Note that changing the group is optional, since it's created in the "Make sure we have a '{{ deploy_group }}' group" step if it doesn't already exist. Maybe we can change the sentence a bit so it's clear this is an optional step :thinking:.

```suggestion
If you are on Ubuntu and would like to use its default `sudo` group instead of `wheel`, change the `deploy_group` variable to:
```

Note: I'm not a native English speaker, so feel free to correct the sentence if there are any mistakes :wink:.

Reply:
Thanks for the quick review.  I did test it on Ubuntu 20.04 without using root access, but with a sudo user, and it worked ok.  Can test it on 18.04. Guess migration to 22.04 will also be required. Moving to the configuration section is probably ok, especially since the reference to the file is there.  Howver, since the first section indicates it should work on both Debian and Ubuntu, it may be helpful to have a pointer about changes required for Ubuntu setup, possibly something similar to `The default user is deploy but you can [use any user](https://github.com/consul/installer#using-a-different-user-than-deploy) with sudo privileges.` which has an appropriate hyperlink.